content/integrations/cloud_foundry.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Datadog-Cloud Foundry Integration
+integration_title: Cloud Foundry
+kind: integration
+git_integration_title: cloud_foundry
+newhlevel: true
+beta: true
+---
+# Overview
+
+Any Cloud Foundry deployment can send metrics and events to Datadog. The data helps you—the deployment administrator—track the health and availability of all nodes in the deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more.
+
+There are three points of integration with Datadog, each of which achieves a different goal:
+
+* **Datadog Agent BOSH release** — Install the Datadog Agent on every node in your deployment to track system, network, and disk metrics. Enable any other Agent checks you wish.
+* **Datadog Firehose Nozzle** — Deploy one or more Datadog Firehose Nozzle jobs. The jobs tap into your deployment's Loggregator Firehose and send all non-container metrics to Datadog.
+* **Datadog plugin for BOSH Health Monitor** — Configure your BOSH Director's Health Monitor to send heartbeats (as metrics) and alerts (as events) from each node's BOSH Agent to Datadog.
+
+If you do NOT administer a Cloud Foundry deployment—i.e. if you're an end user deploying applications onto a Cloud Foundry deployment—this page cannot help you.
```

Comment:
Lets rephrase this as "You must have administrative access to your CloudFoundry cluster in order to deploy these monitoring components."